lotto/client/js/Lottos.js

```diff
@@ -0,0 +1,30 @@
+import {lottoTemplate} from "./templates.js"
+import {LOTTO} from "./constants.js";
+
+export function Lottos() {
+    const $lottoList = document.querySelector("#lottos");
+
+    this.setLottos = ticketCount => {
+        $lottoList.innerHTML = "";
+        for (let i = 0; i < ticketCount; i++) {
+            render(generateLotto());
+        }
+    };
+
+    const generateLotto = () => {
+        let lotto = [];
+        while (lotto.length !== LOTTO.LENGTH) {
+            let randomNumber = Math.floor(Math.random() * LOTTO.MAX_NUMBER + LOTTO.MIN_NUMBER);
+            if (!lotto.includes(randomNumber)) {
+                lotto.push(randomNumber);
+            }
+        }
+        return lotto;
+    };
+
+    const render = lotto => {
+        const template = lottoTemplate(lotto);
+        $lottoList.insertAdjacentHTML("beforeEnd", template);
```

Comment:
String 형식이라 헷갈릴 수 있지만, [API](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML)를 참고해서 정확한 명칭을 써주셔야합니다! 

```suggestion
        $lottoList.insertAdjacentHTML("beforeend", template);
```